micrometer-meter-provider/src/test/java/io/opentelemetry/contrib/metrics/micrometer/internal/PollingMeterCallbackRegistrarTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.metrics.micrometer.internal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import io.opentelemetry.contrib.metrics.micrometer.CallbackRegistration;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PollingMeterCallbackRegistrarTest {
+
+  SimpleMeterRegistry meterRegistry;
+
+  @BeforeEach
+  void setUp() {
+    meterRegistry = new SimpleMeterRegistry();
+  }
+
+  @Test
+  void createsPollingMeterOnCallbackRegistration() {
+    Meter pollingMeter;
+    try (PollingMeterCallbackRegistrar underTest =
+        new PollingMeterCallbackRegistrar(() -> meterRegistry)) {
+
+      pollingMeter = meterRegistry.find("otel_polling_meter").meter();
+      assertThat(pollingMeter).isNull();
+
+      underTest.registerCallback(() -> {});
+
+      pollingMeter = meterRegistry.get("otel_polling_meter").meter();
```

Comment:
`MeterRegistry#get()` will never return a null meter, it will throw instead. For consistency, consider using `find()` everywhere.